source/reference/options.txt

```diff
@@ -393,12 +393,25 @@ Authentication Options
    :option:`--password <--password>` option as the last option and leave out the
    argument.
 
+.. option:: --quiet
+
+   Skip all messages during startup (such as welcome messages and startup
```

Comment:
nit (non-blocking):

I took a look at the other examples and many (though not all) seem to be in the simple present (e.g. "Increases", "Specifies", "Bypasses"). Perhaps it's worthwhile to update the tense here ("Skip" -> "Skips", "go" -> "goes") for consistency. Just a thought!